src/main/kotlin/no/nav/syfo/api/model/SenOppfolgingKandidatResponseDTO.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.syfo.api.model
+
+import no.nav.syfo.domain.SenOppfolgingKandidat
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class SenOppfolgingKandidatResponseDTO(
+    val uuid: UUID,
+    val createdAt: LocalDateTime,
+    val personident: String,
+    val ferdigbehandlet: FerdigbehandletResponseDTO?,
+)
+
+data class FerdigbehandletResponseDTO(
+    val veilederident: String,
+    val createdAt: LocalDateTime,
+)
```

Comment:
Trenger vi å ha en property `ferdigbehandlet` eller kunne vi bare satt den oppdaterte ressursen i helhet som response? På denne måten legger vi opp til hvis en klient har lyst å vise den oppdaterte ressursen eller gjøre noe annet.

Reply:
Ja, du tenker `status` og `vurderinger` property i responsen i stedet? Kanskje ryddigst det ja. Fint om klienten får det på samme format i responsen her og i et framtidig GET-endepunkt for å hente kandidat i hvert fall :+1: